packages/appsync-modelgen-plugin/src/visitors/appsync-swift-visitor.ts

```diff
@@ -305,7 +305,7 @@ export class AppSyncSwiftVisitor<
         .asKind('enum')
         .access('public')
         .withProtocols(['String', 'EnumPersistable'])
-        .withName(this.getEnumName(enumValue));
+        .withName(pascalCase(this.getEnumName(enumValue)));
```

Comment:
Is there a swift test case?

Reply:
Yes - [should generate an enum for enum type](https://github.com/aws-amplify/amplify-codegen/blob/b8df7e6405b7f9888bcbb632702fa9fe43fff980/packages/appsync-modelgen-plugin/src/__tests__/visitors/appsync-swift-visitor.test.ts#L161-L189).